multinode-demo/bootstrap-validator.sh

```diff
@@ -97,9 +97,6 @@ while [[ -n $1 ]]; do
     elif [[ $1 == --no-snapshot-fetch ]]; then
       args+=("$1")
       shift
-    elif [[ $1 == --allow-private-addr ]]; then
-      args+=("$1")
-      shift
```

Comment:
why do we need to remove this flag?
can't the caller set the flag if needed like before?

Reply:
It's possible I'm misunderstanding something, but my thinking is we need `allow-private-addr` to always be true now because otherwise we will filter out loopback IPs, which are actually legitimate in these local cluster environments. So rather than an optional command line flag, it is now always passed in as mandatory.

`--allow-private-addr`--> `SocketAddrSpace::Unspecified` --> loopback IPs allowed

Reply:
> we need `allow-private-addr` to always be true now because otherwise we will filter out loopback IPs, which are actually legitimate in these local cluster environments.

well these scripts are not tied to local cluster environments; e.g. they can be used to deploy a cluster across multiple data centers. We probably need to keep the option to mirror mainnet setup.

Reply:
Makes sense. Reverted those scripts back to accepting the param, passing the param as part of localnet-sanity, and updated documentation for running local multinode test